### ISSUE TITLE
want to save ckpt files to external drive

### ISSUE THREAD

**Issue by ghffarpk** [thtrieu/darkflow#877]
Hi, I am using google colab to train the model. How i can save checkpoint files to google drive. I just need to know, can we give it a custom folder path to save the checkpoint files?

**Comment by sandeeprepakula** [comment]
Yes. Use the command line option --backup $checkpoint_folder_path
